views/block.tmpl

```diff
@@ -1,362 +1,354 @@
+						{{template "decimalParts" (float64AsDecimalParts .Total 8 false)}}
+					</td>
+					<td class="mono fs15 text-right">{{.FormattedSize}}</td>
+				</tr>
+				{{- end}}
+			</tbody>
+		</table>
+		{{- end}}
 
-        {{if ge .Height .StakeValidationHeight}}
-        {{if .Misses}}
-            <span class="d-inline-block pt-4 pb-1 h4">Missed Votes</span>
-            <table class="table">
-                <thead>
-                    <th>Ticket ID</th>
-                </thead>
-                <tbody>
-                    {{range .Misses}}
-                    <tr>
-                        <td class="break-word">
-                            <span>
-                                <a class="hash lh1rem" href="/tx/{{.}}">{{.}}</a>
-                            </span>
-                        </td>
-                    </tr>
-                    {{end}}
-                </tbody>
-            </table>
-        {{end}}
-        {{end}}
+	  {{- if ge .Height .StakeValidationHeight -}}
+	  {{if .Misses -}}
+		<span class="d-inline-block pt-4 pb-1 h4">Missed Votes</span>
+		<table class="table">
+			<thead>
+				<th>Ticket ID</th>
+			</thead>
+			<tbody>
+				{{range .Misses -}}
+				<tr>
+					<td class="break-word">
+						<span>
+							<a class="hash lh1rem" href="/tx/{{.}}">{{.}}</a>
+						</span>
+					</td>
+				</tr>
+				{{- end}}
+			</tbody>
+		</table>
+	  {{- end}}
+	  {{- end}}
 
-            <span class="d-inline-block pt-4 pb-1 h4">Tickets</span>
-            {{if not .Tickets}}
-            <table class="table">
-                <tr>
-                    <td>No tickets mined this block.</td>
-                </tr>
-            </table>
-            {{else}}
-                <table class="table">
-                    <thead>
-                        <th>Transaction ID</th>
-                        <th class="text-right">Total DCR</th>
-                        <th class="text-right">Fee</th>
-                        <th class="text-right">Size</th>
-                    </thead>
-                    <tbody>
-                    {{ range .Tickets}}
-                        <tr>
-                            <td class="break-word">
-                                <span>
-                                    <a class="hash" href="/tx/{{.TxID}}">{{.TxID}}</a>
-                                </span>
-                            </td>
-                            <td class="text-right dcr mono fs15">
-                              {{template "decimalParts" (float64AsDecimalParts .Total 8 false)}}
-                            </td>
-                            <td class="mono fs15 text-right">{{.Fee}}</td>
-                            <td class="mono fs15 text-right">{{.FormattedSize}}</td>
-                        </tr>
-                    {{end}}
-                    </tbody>
-                </table>
-            {{end}}
+		<span class="d-inline-block pt-4 pb-1 h4">Tickets</span>
+	  {{- if not .Tickets}}
+		<table class="table">
```

Comment:
A lot of space-to-tab conversion happening here. Is that the intent?

Reply:
Yeah, but I've not backed that decision up with post-gzip sizes.  Its a lot fewer chars though.
TBH I'll probably abandon this PR, maybe just grabbing the couple of legit fixes for a different PR.